src/helpers/FormioLoader/index.js

```diff
@@ -206,11 +163,4 @@ const initFormio = () => {
   });
 };
 
-// Persistent fix for iPhone/Safari
-window.onpageshow = (event) => {
```

Comment:
where is this going as its not in your defaults

Reply:
i put this piece of code back to squiz matrix, doesn't seems to be a part of the formio related code